DigitalLearningSolutions.Web/Controllers/HomeController.cs

```diff
@@ -57,7 +61,9 @@ public IActionResult Products()
         [HttpGet]
         public IActionResult LearningContent()
         {
-            return View(GetLandingPageViewModel(2));
+            var learningContents = brandsService.GetPublicBrandsDetails()
+                .Select(b => new LearningContentSummary(b));
+            return View(GetLearningContentLandingPageViewModel(learningContents));

```

Comment:
I think it would be a little more readable if we create a variable `model` for the view model and pass it to the View